Cargo.toml

```diff
@@ -59,7 +59,7 @@ default-features = false
 
 
 [dev-dependencies]
-cortex-m-rtic = "0.5.5"
+cortex-m-rtic = "0.6.0-alpha.0"
```

Comment:
Since this an alpha version, this should be an exact match, since semver guarantees don't apply

Reply:
Thanks! Somehow I had manged to completely forget this again.